packages/circle-demo-webapp/app/routes/wallet.$id/components/WalletAddressDialog.tsx

```diff
@@ -4,25 +4,25 @@ import { Wallet } from 'web3-circle-sdk';
 
 import { Button } from '~/components/ui/button';
 import { Dialog, DialogContent, DialogTrigger } from '~/components/ui/dialog';
-import { WalletReceive } from '~/components/WalletReceive';
+import { WalletAddressHelpers } from '~/components/WalletAddressHelpers';
 
-interface WalletReceiveDialogProps {
+interface WalletAddressDialogProps {
   wallet: Wallet;
 }
 
-export function WalletReceiveDialog({ wallet }: WalletReceiveDialogProps) {
+export function WalletAddressDialog({ wallet }: WalletAddressDialogProps) {
```

Comment:
I think here you went to far with the renaming as the reusable component name is one thing (as explained in my other comment) and the context it is used is another thing. And in this case the component is used in WalletReceiveDialog as it's purpose is display wallet details to receive funds. The name is also connected to the label of the button that triggers the feature. wdyt?

Reply:
All good...reverted 💯